compiler/optimizer/Dominators.cpp

```diff
@@ -459,14 +457,15 @@ void TR_PostDominators::findControlDependents()
    int32_t nextNodeNumber = _cfg->getNextNodeNumber();
    int32_t i;
 
+   _directControlDependents = (TR_BitVector**)_region.allocate(nextNodeNumber*sizeof(TR_BitVector *));
```

Comment:
I'm not sure whether the in place value TR_BitVector will work nicely with all the allocator copying? This mathces how a number of data structures are declared in the optimizer so I was following that pattern. We could consider reworking it as you suggest in the future.